doc/sphinx/addendum/type-classes.rst

```diff
@@ -349,10 +348,10 @@ Summary of the commands
 
 .. cmd:: Instance {? @ident_decl {* @binder } } : @type {? @hint_info } {? {| := %{ {* @field_val } %} | := @term } }
 
-   Declares a typeclass instance named
-   :token:`ident_decl` of the class :n:`@type` with the specified parameters and with
-   fields defined by :token:`field_val`, where each field must be a declared field of
-   the class.
+   Declares a typeclass instance named :token:`ident_decl` of the typeclass
+   :n:`@type` with the specified parameters and with fields defined by
+   :token:`field_val`, where each field must be a declared field of the
+   typeclass.
```

Comment:
```suggestion
   Declares the typeclass instance :token:`ident_decl` of the type
   :n:`@type` with the specified parameters and with fields defined by
   :token:`field_val`, where each field must be a declared field of the
   typeclass.
```

typeclass instance is different from a typeclass?

"where each field must be a declared field of the typeclass." - or of the type?

Reply:
Yes, the terminology borrows from OOP (think Java) where you have "Classes" and "Instances" of those classes.

Reply:
I will keep a as a class may have many instances.